src/tools/optimization-options.h

```diff
@@ -39,7 +39,7 @@ struct OptimizationOptions : public ToolOptions {
     (*this)
       .add("",
            "-O",
-           "execute default optimization passes",
+           "execute default optimization passes (equivalent to -Os)",
```

Comment:
I worry this might get stale if we change the default but forget to change this help text. How about adding some assertions to verify that, say after line 46 where we set the default opts, we can check that `optimizeLevel == 2, shrinkLevel == 1`?

Reply:
Good point, how is the new version?
With the static constants in `OptimizationOptions`, we also catch changes that are done on `-Os`'s side.
Added another commit also introducing static constants with the default options to `PassOptions`, in order to make the assertion a `static_assert`. How is that? I suggest I would squash that second commit if you like that solution and delete it otherwise.

Reply:
(side note, about landing, we squash all PRs anyhow, to guarantee the main branch is 100% bisectable. so no need to worry about the commits in the PR, that is.)